spec.bs

```diff
@@ -569,12 +583,16 @@ steps. They return a [=boolean=]:
     not null, return true.
 1. If |preSpecifiedParams|' [=pre-specified report parameters/filtering ID max
     bytes=] is not the [=default filtering ID max bytes=], return true.
+1. If |preSpecifiedParams|' [=pre-specified report parameters/requested max
+    contributions=] is not null, return true.
 1. Return false.
 
-Note: If a context ID or non-default filtering ID max bytes was specified, a
-    report is sent, even if there are no contributions or there is insufficent
-    budget for the requested contributions. See [Protecting against leaks via
-    the number of reports](#protecting-against-leaks-via-the-number-of-reports).
+Note: Even when budget is insufficient for the requested contributions, a report
```

Comment:
Should we include the "no contributions requested" case in this note?

Reply:
Sure. I'll give this another pass -- let me know if you preferred the original.